app/controllers/manage_cmr/order_options_controller.rb

```diff
@@ -54,30 +131,95 @@ def create
     end
   end
 
-  def show
+  def deprecate_order_option
```

Comment:
I think a comment may be needed something along the lines of "deprecate AKA remove order option" or something to that affect. To the uninitiated it looks like you renamed a function you want to delete latter. Everyone involved probably understands you, but I'm sure someone in the future will be confused.

Reply:
No, this function is to do deprecating an order option (not it self is deprecated)